libs/coin-modules/coin-evm/src/bridge/getTransactionStatus.ts

```diff
@@ -115,7 +115,7 @@ const validateAmount = (
   // if no amount or 0
   if ((!transaction.amount || transaction.amount.isZero()) && !canHaveZeroAmount) {
     errors.amount = new AmountRequired(); // "Amount required"
-  } else if (totalSpent.isGreaterThan(account.balance)) {
+  } else if (totalSpent.isGreaterThan(account.balance) && !transaction.sponsored) {
     // if not enough to make the transaction
     errors.amount = new NotEnoughBalance(); // "Sorry, insufficient funds"
   }
```

Comment:
I'm not sure we just want to allow any transaction spending more than the balance but only the ones that would not cover the fees

Or even better the totalSpent should be correct for sponsored transactions

Reply:
You would need to allow sending 0 for `sponsored` transactions because we have a check for it too I think
I think it's better if we don't add fees to the amount if `sponsored` is `true`